content/protobuf/language-mapping/rpc-method.md

```diff
@@ -0,0 +1,70 @@
+---
+title: RPC method
+description: Learn how RPC methods are mapped to C#.
```

Comment:
```suggestion
description: How RPC methods are mapped to C#.
```

We use "learn" in lots of descriptions. I think a lot of the time we can simplify by just removing learn.